tests/bsim/bluetooth/audio_samples/bap_broadcast_sink/tests_scripts/broadcast_audio_interleaved.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Copyright 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# Simple selfchecking test for the broadcast audio sink/source samples,
+# It relies on the bs_tests hooks to register a test timer callback, which after a deadline
+# will check how many audio packets the broadcast audio sink has received, and if over a threshold
+# it considers the test passed
+
+simulation_id="samples_bluetooth_bap_broadcast_source_interleaved_test"
+sample_long_name="samples_bluetooth_bap_broadcast_source"
```

Comment:
Why are you defining this as a variable if only used one place?
Why are you not defining this for the broadcast sink side?

Reply:
The built executable name is longer than 100 columns with `_overlay-interleaved_conf` being appended. Seems thats not the case in the latest upstream/main after 1bcc3d8efa1. updated.